scripts/proposals/AIP4/data/421613-AIP4-data.json

```diff
@@ -0,0 +1,9 @@
+  "arbSysSendTxToL1Args": {
```

Comment:
would be cool if we could also include in this data json the calldata that is expected by the L2 Governor (like target address being ArbSys, and the calldata to it

Reply:
can make this change in a future PR tho